packages/neos-ui-sagas/src/CR/NodeOperations/helpers.js

```diff
@@ -1,15 +1,5 @@
 import {findNodeInGuestFrame, closestNodeInGuestFrame} from '@neos-project/neos-ui-guest-frame/src/dom';
 
-export const parentNodeContextPath = contextPath => {
```

Comment:
that's the 2nd occurence of client-side parent calculation which we need to get rid of